gateway/stateful-gateway/src/main/java/com/jdriven/gateway/stateful/TestController.java

```diff
@@ -0,0 +1,22 @@
+package com.jdriven.gateway.stateful;
+
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.security.oauth2.client.authentication.OAuth2AuthenticationToken;
+import org.springframework.security.oauth2.core.user.OAuth2User;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+@RestController
+public class TestController {
```

Comment:
Misschien aardig vast te leggen in comments/JavaDoc wat deze controller toevoegt..
En of het echt nodig is voor functioneren of niet.